addons/project/models/project_project.py

```diff
@@ -1140,3 +1148,106 @@ def _thread_to_store(self, store: Store, fields, *, request_list=None):
     def _compute_task_completion_percentage(self):
         for task in self:
             task.task_completion_percentage = task.task_count and 1 - task.open_task_count / task.task_count
+
+    # ---------------------------------------------------
+    #  Project Template Methods
+    # ---------------------------------------------------
+
+    def _is_project_linked(self):
```

Comment:
the naming does not seem good because with that name we could expect to only have boolean value as return value but it is not really the case.